### ISSUE TITLE
[dst] make test scripts that require ROOT optional (Trac #1639)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1631]
Both test scripts, `dst13.py` and `dst16.py`, call the corresponding DST extractors, which require ROOT. Because ROOT is optional, the tests should also be declared as optional.

Migrated from https://code.icecube.wisc.edu/ticket/1639
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Both test scripts, `dst13.py` and `dst16.py`, call the corresponding DST extractors, which require ROOT. Because ROOT is optional, the tests should also be declared as optional.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[dst] make test scripts that require ROOT optional", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-04-17T15:26:45", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160417T16:48:05: olivas changed milestone from "" to "IceRec V5"

**Comment by icecube-trac** [comment]
Trac update at 20160417T16:48:28: 
* olivas changed component from "simulation" to "icerec"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160503T13:28:24: 
* olivas changed status from "assigned" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 145378:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145378"
> only run the tests if ROOT is found and used. closes #1631.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
